metadata-ingestion/tests/unit/test_sagemaker_source.py

```diff
@@ -0,0 +1,73 @@
+import json
+
+from botocore.stub import Stubber
+from freezegun import freeze_time
+
+from datahub.ingestion.api.common import PipelineContext
+from datahub.ingestion.source.sagemaker import SagemakerSource, SagemakerSourceConfig
+from tests.test_helpers import mce_helpers
+from tests.unit.test_sagemaker_source_stubs import (
+    describe_feature_group_response_1,
+    describe_feature_group_response_2,
+    describe_feature_group_response_3,
+    list_feature_groups_response,
+)
+
+FROZEN_TIME = "2020-04-14 07:00:00"
+
+
+def sagemaker_source() -> SagemakerSource:
+    return SagemakerSource(
+        ctx=PipelineContext(run_id="sagemaker-source-test"),
+        config=SagemakerSourceConfig(aws_region="us-west-2"),
+    )
+
+
+@freeze_time(FROZEN_TIME)
+def test_sagemaker_ingest(tmp_path, pytestconfig):
```

Comment:
probably makes sense to keep this in the `tests/unit/sagemaker` directory as well